src/test/java/org/folio/rest/api/LoanPoliciesApiTest.java

```diff
@@ -314,6 +314,93 @@ public void cannotCreateALoanPolicyWithAdditionalPropertiesInLoanPolicy()
       response.getStatusCode(), is(HttpStatus.HTTP_VALIDATION_ERROR.toInt()));
   }
 
+  @Test
+  public void canCreateLoanPolicyWithItemLimitWithinBounds()
+    throws MalformedURLException,
+    InterruptedException,
+    ExecutionException,
+    TimeoutException {
+
+    CompletableFuture<JsonResponse> createCompleted = new CompletableFuture<>();
+
+    UUID id = UUID.randomUUID();
+
+    JsonObject loanPolicyRequest = defaultRollingPolicy()
+      .withId(id)
+      .withName("Example Loan Policy")
+      .withDescription("An example loan policy")
+      .create();
+
+    loanPolicyRequest.getJsonObject("loansPolicy").put("itemLimit", 1000);
+
+    client.post(loanPolicyStorageUrl(),
+      loanPolicyRequest, StorageTestSuite.TENANT_ID,
+      ResponseHandler.json(createCompleted));
+
+    JsonResponse response = createCompleted.get(5, TimeUnit.SECONDS);
+
+    assertThat(String.format("Failed to create loan policy: %s", response.getBody()),
```

Comment:
Could you please add an assertion to check that the property is really got persisted (i.e. it returned in post response, or read the policy by ID and verify that the property is present)?

Reply:
done